### DISCUSSION TITLE
Connecting Azure storage as backend and hosting in Azure webApp

### DISCUSSION THREAD

**Discussion by ghoshdipanjan** [photoview/photoview#437] [General]
Just wanted to understand couple of things:
- Can we host this straight in an Azure webapp(it supports docker containers) without any change and is this been tried before?
- Instead of a local path, can we pass it a connection string or an url path to a storage in cloud?

Would be good to know if these scenarios are possible, if any one has done it, happy to know the "How Tos"!

**Comment by viktorstrate** [comment]
I've never used Azure before, but if it supports Docker, I can't see why it wouldn't work.

Photoview doesn't support remote storage in it of itself. You'll have to configure that outside of the container and use a Docker bind mount to access it. But Azure might have a way to do that?
Maybe something like this: https://azure.microsoft.com/da-dk/blog/persistent-docker-volumes-with-azure-file-storage/

**Reply by ghoshdipanjan** [reply]
Thanks Victor, that clarifies a lot of questions. In terms of storage, yes we should be able to mount a drive and simulate it, although the fun would be to have lazy read from any blob storage via apis.

Thanks for taking time answer.
